spinny_gun.py

```diff
@@ -478,6 +525,8 @@ def game_loop():
             if missile.rect[1] > DISPLAY_HEIGHT - missile.image.get_height():
                 missiles.pop(missiles.index(missile))
                 player.update_health(-1)
+                if player.health < 1:
```

Comment:
```suggestion
                if player.health <= 0:
```

I feel like this makes more sense